shared/settings/advanced.tsx

```diff
@@ -82,17 +82,19 @@ const Advanced = () => {
 
   // load it
   if (disableSpellCheck === undefined) {
-    loadDisableSpellcheck(
-      [{path: 'ui.disableSpellCheck'}],
-      result => {
-        const res = result.b ?? false
-        initialDisableSpellCheck.current = res
-        setDisableSpellcheck(res)
-      },
-      () => {
-        setDisableSpellcheck(false)
-      }
-    )
+    setTimeout(() => {
```

Comment:
should likely always do this at the framework level but, too scary a change for now